### DISCUSSION TITLE
Add install package at new project stage

### DISCUSSION THREAD

**Discussion by gshotwell** [posit-dev/positron#5233] [Feedback]
I use a distinct venv for each project, but I often find myself installing the same packages when I get started. It might be nice to allow the user to select packages they'd like to install when creating a new virtual environment project, this could remember the packages that you  selected last time or refer to a user setting.

**Comment by juliasilge** [comment]
We have https://github.com/posit-dev/positron/issues/1529 where we talk about letting folks use, say, a `requirements.txt` plus a Python version as a template for new projects. This is an interesting related idea!
